lib/solr_batch.rb

```diff
@@ -0,0 +1,41 @@
+require "clusterable/holding"
+require "solr_record"
+
+# A chunk of solr records for which we can fetch holdings in a batch
```

Comment:
Needs more documentation

Reply:
@moseshll I think this is a bit more comprehensible than before but let me know what you think here

Reply:
Yes this makes sense.